src/sinks/nats.rs

```diff
@@ -83,6 +83,10 @@ impl SinkConfig for NatsSinkConfig {
     fn sink_type(&self) -> &'static str {
         "nats"
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
```

Comment:
I think this can be true. The library we use returns an async Result when sending so you should be able to ack when that comes back.

Reply:
I don't think we actually do, though. I see no evidence in the code that we actually act on the acknowledgement to set an `EventStatus`. Did I miss it?

Reply:
Yeah, I'm not seeing it either 🤔 

This sink certainly _could_ support acknowledgements.